### PR TITLE
Update high-elf-realms.json

### PR DESCRIPTION
**Pull request by Kelthraii**
Added Spanish language

### CODE REVIEW

**Comment by nthiebes** [comment]
Awesome, thanks! I just pushed some changes I made earlier so once the conflict is resolved we can merge 👍

**Comment by nthiebes** [comment]
@Kelthraii ping :)

**Comment by Kelthraii** [comment]
On it!

**Comment by Kelthraii** [comment]
Done! Sorry to keep you waiting, I've been busy with work and I had to do this twice, as I left the tab open for too long and it refreshed eventually, erasing all I had done before :(
But I think everything is more or less tidy now, there was French everywhere!

**Review by nthiebes — COMMENTED** [review]
No worries, I'm also way too busy right now 🙈

File: public/games/the-old-world/high-elf-realms.json [hunk 1]
```diff
@@ -400,94 +445,101 @@
...
               "name_en": "Eagle-Eye Bolt Thrower",
               "name_it": "Eagle-Eye Balista",
               "name_de": "Eagle-Eye Bolt Thrower",
+              "name_fr": "Baliste Œil-d'aigle",
+              "name_es": "Lanzavirotes ojo de águila",
               "points": 25,
-              "name_fr": "Baliste Œil-d'aigle"
             }
           ],
-          "name_fr": "Cotre Volant de Lothern"
```
[the remark this answers is not in the document]
Comment (Kelthraii) [reply public/games/the-old-world/high-elf-realms.json:441]: It's up there in line 480

**Pull request merged**